tv.plex.PlexHTPC.yml

```diff
@@ -0,0 +1,109 @@
+app-id: tv.plex.PlexHTPC
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: /app/Plex.sh
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --device=all
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-music:ro
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-download:create
+  - --filesystem=xdg-cache:rw
+  - --filesystem=xdg-config:rw
+  - --filesystem=xdg-data:rw
+  - --filesystem=home:ro
+  - --talk-name=org.freedesktop.PowerManagement
+  - --talk-name=org.freedesktop.ScreenSaver
+modules:
+  - name: plex-icons
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 $FLATPAK_ID.desktop /app/share/applications/$FLATPAK_ID
+      - install -Dm664 $FLATPAK_ID.png /app/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
+    sources:
+      - type: dir
+        path: .
+
+  - name: libjpeg # with libjpeg.so.8
+    buildsystem: cmake-ninja
+    config-opts: &libjpeg_config_opts
+      - -DCMAKE_SKIP_RPATH:BOOL=YES
+      - -DENABLE_STATIC:BOOL=NO
+      - -DWITH_JPEG8:BOOL=YES
+      - -DCMAKE_INSTALL_LIBDIR=/app/lib # uses lib64 by default
+    sources: &libjpeg_sources
+      - type: archive
+        url: https://github.com/libjpeg-turbo/libjpeg-turbo/archive/refs/tags/2.1.3.tar.gz
+        sha256: dbda0c685942aa3ea908496592491e5ec8160d2cf1ec9d5fd5470e50768e7859
+
+  - name: libevent
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DEVENT__LIBRARY_TYPE=SHARED
+    sources:
+      - type: git
+        url: https://github.com/libevent/libevent/
+        branch: release-2.1.12-stable
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /lib/cmake
+
+  - name: libwebp
```

Comment:
There is libwebp 1.2.1 in the runtime

Reply:
As discussed in https://github.com/flathub/flathub/pull/3224#discussion_r886928228